akka-stream-tests/src/test/scala/akka/stream/io/OutputStreamSourceSpec.scala

```diff
@@ -147,5 +148,13 @@ class OutputStreamSourceSpec extends AkkaSpec(UnboundedMailboxConfig) {
       sourceProbe.expectMsg(GraphStageMessages.DownstreamFinish)
       the[Exception] thrownBy outputStream.write(bytesArray) shouldBe a[IOException]
     }
+
+    "fail to materialize with zero sized input buffer" in {
+      an[IllegalArgumentException] shouldBe thrownBy {
+        StreamConverters.asOutputStream(timeout)
+          .withAttributes(inputBuffer(0, 0))
+          .runWith(Sink.head)
```

Comment:
As above, test would pass with `Sink.ignore` even without fixes to OutputStreamSource, so i used Sink.head.